Assets/Scripts/Gameplay/Action/BaseTypes/Action.cs

```diff
@@ -24,54 +30,97 @@ namespace Unity.Multiplayer.Samples.BossRoom.Actions
     ///
     /// Note also that if Start() returns false, no other functions are called on the Action, not even End().
```

Comment:
reminder for note about action system not be meant to be reused in projects as is.

Reply:
done!